tensorflow_data_validation/utils/test_util.py

```diff
@@ -29,6 +30,41 @@
 from tensorflow_metadata.proto.v0 import statistics_pb2
 
 
+def _make_example_dict_equal_fn(
```

Comment:
Can you make this a global function (remove underscore)?

```suggestion
def make_example_dict_equal_fn(
```